airbyte-webapp/src/pages/routes.tsx

```diff
@@ -155,6 +155,7 @@ export const Routing = () => {
         <SupportChat
           papercupsConfig={config.papercups}
           customerId={workspace.customerId}
+          onClick={() => (window as any).open(config.ui.slackLink, "_blank")}
```

Comment:
Won't adding `open` function for Window interface in `config.ts` fix this problem with typings?

Reply:
Fixed